src/SFA.DAS.ApplyService.Data/ClarificationRepository.cs

```diff
@@ -164,5 +169,21 @@ public async Task SubmitClarificationPageOutcome(Guid applicationId, int sequenc
                 */
             }
         }
+
+        public async Task DeleteClarificationFile(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId, string clarificationFile)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                await connection.ExecuteAsync(
+                    @"UPDATE [ModeratorPageReviewOutcome]
+			            SET [ClarificationFile] = NULL
```

Comment:
I think you might want to update the ClarificationUpdatedAt to UtcNow as well?

Reply:
done 👍